pkgs/data/fonts/iosevka/bin.nix

```diff
@@ -12,7 +12,7 @@ in fetchzip {
     unzip -j $downloadedFile \*.ttc -d $out/share/fonts/iosevka
   '';
 
-  sha256 = "1dfm1888rii5kfmkxp5hnx8ycji57cbs5gazpgkxg1mnmn7i35wl";
+  sha256 = "1hpwf4nyjl8rxwjiwmva07c0mn0pqyydxnzsvrzavydjivfqykzk";
```

Comment:
hash mismatch in fixed-output derivation '/nix/store/kzdqlrvvcvk15ddzg7jlgwyvwmp2qlpy-iosevka-bin-3.1.0':
  wanted: sha256:1hpwf4nyjl8rxwjiwmva07c0mn0pqyydxnzsvrzavydjivfqykzk
  got:    sha256:1mnf6yp0v1svkhjf598yhcbihhd150nxrddk14d84ardk0d6z3na

Reply:
Oh, I ran `nix-prefetch-url` incorrectly. Thanks for the catch!